javascript/javascript1/week2/Flight booking fullname/fullname.js

```diff
@@ -0,0 +1,19 @@
+let userFirstName = prompt("Enter your first name : ");
+let userSurName = prompt("Enter your Sur Name : ");
+let useFormalName = prompt("Do you want to use your formal name? (yes or no)");
+let userGender = prompt("Let me know your gender ? (male or female)");
+
+function getFullName(firstName, surName, formalName, gender) {
```

Comment:
You can try and refacgtor this function with using only if statement + return.
Somehting like this: 
```
if(someCondition) { 
return someValue; } 

if(someOtherCondition) { 
return someOtherValue; } 

//default case 
return defaultValue;
```
it usually makes the function easier to read